rpc_jobs/standard_job_l2_gate.yml

```diff
@@ -0,0 +1,77 @@
+- project:
+    name: "l2_gate_tests"
```

Comment:
I'd also flip this name to something like `rpc-product-1-gate` as the project names will have to be unique across all components.

Reply:
Done